.eslintrc

```diff
@@ -6,10 +6,10 @@
   "parser": "@typescript-eslint/parser",
   "plugins": ["react-hooks", "testing-library"],
   "extends": [
+    "eslint:recommended",
     "plugin:react/recommended",
-    "plugin:prettier/recommended",
     "plugin:testing-library/react",
-    "eslint:recommended"
+    "plugin:prettier/recommended"
```

Comment:
I don't think it was affecting anything, bu https://github.com/prettier/eslint-plugin-prettier#recommended-configuration says prettier/recommended should be the last extension.